pkg/daemon/ceph/client/pool.go

```diff
@@ -273,6 +273,14 @@ func setCommonPoolProperties(context *clusterd.Context, clusterInfo *ClusterInfo
 		if err != nil {
 			return errors.Wrapf(err, "failed to enable mirroring for pool %q", poolName)
 		}
+
+		// Schedule snapshots
+		if pool.Mirroring.SnapshotSchedule.Enabled() {
```

Comment:
Can any pool be enabled for snapshot scheduling (including cephfs/rgw pools)?

Reply:
Potentially yes, but I'd like to stick to the RBD use case.

Rgw has a multi-site already is replication is needed.
Cephfs has nothing.